packages/api/src/controllers/asset.ts

```diff
@@ -706,6 +719,14 @@ app.get("/", authorizer({}), async (req, res) => {
     res.links({ next: makeNextHREF(req, newCursor) });
   }
 
+  if (req.user.admin && deleting) {
+    output = output.filter((asset) => asset.status.phase == "deleting");
+    output = output.filter(
+      (asset) =>
+        asset.deletedAt && Date.now() > asset.deletedAt + DELETE_ASSET_DELAY
+    );
+  }
```

Comment:
**Blocking:** We must filter assets on the SQL query instead, [like this](https://github.com/livepeer/studio/pull/2092/files#diff-ed7a14d016bf3f1e0d94b5823604fde2f1dba33ce1376c00c1237dfa3ed3775bL647), not filtering the DB response in memory.